autrainer/core/utils/bookkeeping.py

```diff
@@ -101,6 +101,76 @@ def create_folder(self, folder_name: str, path: str = "") -> None:
             exist_ok=True,
         )
 
+    def save_dataset_summary(
+        self,
+        data: "AbstractDataset",
+        filename: str,
+        val_exists: bool = False,
+    ) -> None:
+        """Save some overview of the dataset to a file.
+
+        Args:
+            data: dataset object
+            filename: Name of the file to save the overview to.
+        """
+        if val_exists:
+            split_datasets = [data.train_dataset, data.val_dataset, data.test_dataset]
+            index = ["Train", "Development", "Test", "Total"]
+        else:
+            split_datasets = [data.train_dataset, data.test_dataset]
+            index = ["Train", "Test", "Total"]
+        n_samples = []
+        if isinstance(data, BaseClassificationDataset) or isinstance(data, BaseMLClassificationDataset):
+            n_classes = []
+            n_majority_class = []
+            n_minority_class = []
+            class_imbalance = []
+        elif isinstance(data, BaseRegressionDataset):
```

Comment:
We also support multi-target regression now (`BaseMTRegressionDataset`).

Reply:
Ah i missed that this is already supported, anyways, there maybe it would make sense to do the same as for single target regression but for all targets separately?

Reply:
Yep this should be good.